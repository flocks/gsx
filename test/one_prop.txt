test/samples/file.tsx:5:<Button variant="primary" size="large">
--
test/samples/file.tsx:9:<Button variant="secondary">
--
