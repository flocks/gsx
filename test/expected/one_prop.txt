test/samples/file.tsx:5:0:<Button variant="primary" size="large">
--
test/samples/file.tsx:9:0:<Button variant="secondary">
--
